scrapers/freshmaker.py

```diff
@@ -0,0 +1,84 @@
+# SPDX-License-Identifier: GPL-3.0+
+from __future__ import unicode_literals
+import urllib3
+
+from scrapers.base import BaseScraper
+from purview.models.freshmaker import FreshmakerEvent, ContainerBuilds
+from scrapers.utils import retry_session
+from purview.utils.general import timestamp_to_datetime
+from purview import log
+
+urllib3.disable_warnings()
+
+
+class FreshmakerScraper(BaseScraper):
+
+    def run(self, since=None):
+        """
+        Main function that runs the Freshmaker scraper
+        :param since: a string representing a datetime to start scraping data from
+        :return: None
+        """
+        if since:
+            log.info('Start date: {0} does not apply to Freshmaker scraper!'.format(since))
+        log.info('Starting initial load of Freshmaker events')
+        # Initialize session and base url
+        session = retry_session()
+        fm_url = self.freshmaker_url
+        self.query_api_and_update_neo4j(session, fm_url)
+        log.info('Initial load of Freshmaker events complete!')
+
+    def query_api_and_update_neo4j(self, session, fm_url):
```

Comment:
The session could actually be defined in this function so that you don't have to pass it in as a parameter.